pkg/tensorflow-lite/Makefile.include

```diff
@@ -0,0 +1,5 @@
+INCLUDES += -I$(PKGDIRBASE)/tensorflow-lite
+
+ifeq (,$(filter arch_esp8266,$(FEATURES_USED)))
+  CXXEXFLAGS += -Wno-unused-const-variable
```

Comment:
Can you add a comment for this.

Reply:
Apparently, this is not needed anymore (probably because flatbuffers depends on cpp11 explicitly) => removed.